cmd/lava/internal/help/helpdoc.go

```diff
@@ -117,7 +120,7 @@ The sample below is a full agent configuration:
 	  registries:
 	    - server: example.com
 	      username: user
-	      password: p4ssw0rd
+	      password: ${MY_PASSWORD}
```

Comment:
I would use a more formal name for the env var.

```suggestion
	      password: ${REGISTRY_PASSWORD}
```

Reply:
done